src/ia-dropdown.ts

```diff
@@ -194,11 +267,35 @@ export class IaDropdown extends LitElement {
     ul.dropdown-main {
       position: absolute;
       list-style: none;
-      margin: 5px 0 0 0;
+      margin: var(--dropdownOffsetTop, 5px) 0 0 0;
       padding: 0;
       color: var(--dropdownTextColor, #fff);
-      border-radius: 4px;
-      border: 1px solid var(--dropdownBorderColor, #fff);
+      font-size: var(--dropdownFontSize, 1rem);
+      border-radius: var(
+          --dropdownBorderTopLeftRadius,
+          var(--dropdownBorderRadius, 4px)
+        )
+        var(--dropdownBorderTopRightRadius, var(--dropdownBorderRadius, 4px))
+        var(--dropdownBorderBottomRightRadius, var(--dropdownBorderRadius, 4px))
+        var(--dropdownBorderBottomLeftRadius, var(--dropdownBorderRadius, 4px));
+      border-top: var(--dropdownBorderTopWidth, var(--dropdownBorderWidth, 1px))
+        solid var(--dropdownBorderColor, #fff);
+      border-right: var(
+          --dropdownBorderRightWidth,
+          var(--dropdownBorderWidth, 1px)
+        )
+        solid var(--dropdownBorderColor, #fff);
+      border-bottom: var(
+          --dropdownBorderBottomWidth,
+          var(--dropdownBorderWidth, 1px)
+        )
+        solid var(--dropdownBorderColor, #fff);
+      border-left: var(
+          --dropdownBorderLeftWidth,
+          var(--dropdownBorderWidth, 1px)
+        )
+        solid var(--dropdownBorderColor, #fff);
+      white-space: var(--dropdownWhiteSpace, normal);
```

Comment:
Needless to say, Prettier is not being super helpful/consistent here...

Reply:
I may pull some of these fallbacks out into separate consts to make this read a bit nicer.

Reply:
ok, also kinda like the var-with-default-as-default 😅